sympy/polys/series/base.py

```diff
@@ -136,3 +136,91 @@ def square(self, s: TSeries, /) -> TSeries:
     def truncate(self, s: TSeries, n: int, /) -> TSeries:
         """Truncate a power series to the first n terms."""
         ...
+
+
+class PowerSeriesRingProtoZZ(PowerSeriesRingProto[TSeries, Er], Protocol[TSeries, Er]):
+    """A protocol for a power series ring over ZZ."""
+
+    def __call__(self, coeffs: Sequence[Er | int], prec: int | None = None) -> TSeries:
+        """Return a power series element over ZZ."""
+        ...
+
+    def compose(self, s1: TSeries, s2: TSeries, /) -> TSeries:
+        """Compose two power series."""
+        ...
```

Comment:
Are these methods like `compose` or `inverse` particular to ZZ rather than just something that any kind of series would have?

Reply:
Maybe these methods in the ZZ protocol can move to the superclass.